doc/modules/classification_threshold.rst

```diff
@@ -0,0 +1,166 @@
+.. currentmodule:: sklearn.model_selection
+
+.. _tunedthresholdclassifier:
+
+======================================================
+Tuning cut-off decision threshold for class prediction
+======================================================
+
+Classifiers are predictive models: they use statistical learning to predict outcomes.
+The outcomes of a classifier are scores for each sample in relation to each class and
+categorical prediction (class label). Scores are obtained from :term:`predict_proba` or
+:term:`decision_function`. The former returns posterior probability estimates for each
+class, while the latter returns a decision score for each class. The decision score is a
+measure of how strongly the sample is predicted to belong to the positive class (e.g.,
+the distance to the decision boundary). In binary classification, a decision rule is
+then defined by thresholding the scores, leading to a single class label for each
+sample. Those labels are obtained with :term:`predict`.
+
+For binary classification in scikit-learn, class labels are obtained by associating the
+positive class with posterior probability estimates greater than 0.5 (obtained with
+:term:`predict_proba`) or decision scores greater than 0 (obtained with
+:term:`decision_function`).
+
+Here, we show an example that illustrates the relation between posterior
+probability estimates and class labels::
+
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.tree import DecisionTreeClassifier
+    >>> X, y = make_classification(random_state=0)
+    >>> classifier = DecisionTreeClassifier(max_depth=2, random_state=0).fit(X, y)
+    >>> classifier.predict_proba(X[:4])
+    array([[0.94     , 0.06     ],
+           [0.94     , 0.06     ],
+           [0.0416..., 0.9583...],
+           [0.0416..., 0.9583...]])
+    >>> classifier.predict(X[:4])
+    array([0, 0, 1, 1])
+
+While these approaches are reasonable as default behaviors, they are not ideal for
+all cases. Let's illustrate with an example.
+
+Let's consider a scenario where a predictive model is being deployed to assist medical
+doctors in detecting tumors. In this setting, doctors will be most likely interested in
+correctly identifying all patients with cancer so that they can provide them with the
+right treatment. In other words, doctors prioritize achieving a high recall rate,
+meaning they want to identify all cases of cancer without missing any patients who have
+it. This emphasis on recall comes, of course, with the trade-off of potentially more
```

Comment:
```suggestion
identifying all patients with cancer and not missing anyone with cancer so that they can provide them with the
right treatment. In other words, physicians prioritize achieving a high recall rate. This emphasis on recall comes, of course, with the trade-off of potentially more
```